libkbfs/path.go

```diff
@@ -46,21 +49,29 @@ func BuildCanonicalPath(pathType PathType, paths ...string) string {
 	return strings.Join(pathElements, "/")
 }
 
-// buildCanonicalPathForTlfName returns a canonical path for a tlf.
-func buildCanonicalPathForTlfName(public bool, tlfName CanonicalTlfName) string {
-	pathType := PrivatePathType
-	if public {
+func buildCanonicalPathForTlfType(t tlf.Type, paths ...string) string {
+	var pathType PathType
+	switch t {
+	case tlf.Private:
+		pathType = PrivatePathType
+	case tlf.Public:
 		pathType = PublicPathType
+	case tlf.SingleTeam:
```

Comment:
I can't find a way to comment there, but should there be an added case at https://github.com/keybase/kbfs/pull/995/files#diff-98b8e5fde78a630e2ca6a9b7ef015871R42 ?